Add patch to to correct the source to be compatible with gcc6 by
preventing c++ scope errors.

--- mozilla/nsprpub/config/make-system-wrappers.pl	2016-05-12 19:10:05.000000000 +0200
+++ mozilla/nsprpub/config/make-system-wrappers.pl	2016-09-17 15:03:53.901400739 +0200
@@ -19,7 +19,9 @@
     open OUT, ">$output_dir/$_";
     print OUT "#pragma GCC system_header\n";  # suppress include_next warning
     print OUT "#pragma GCC visibility push(default)\n";
+    print OUT "#define _GLIBCXX_INCLUDE_NEXT_C_HEADERS\n";
     print OUT "#include_next \<$_\>\n";
+    print OUT "#undef _GLIBCXX_INCLUDE_NEXT_C_HEADERS\n";
     print OUT "#pragma GCC visibility pop\n";
     close OUT;
 }
